### PR TITLE
CB-17926 Fix logging autoscaling request and flow creation

### PR DESCRIPTION
**Pull request by foldik**
See detailed description in the commit message.

### CODE REVIEW

File: flow/src/main/java/com/sequenceiq/flow/domain/FlowChainLog.java [hunk 1]
```diff
@@ -82,6 +89,13 @@ public String getChain() {
         return chain;
     }
 
+    public Queue<Selectable> getChainAsQueue() {
+        if (StringUtils.isEmpty(chain)) {
+            return new ConcurrentLinkedDeque<>();
+        }
+        return (Queue<Selectable>) JsonReader.jsonToJava(chain);
```
Comment (sodre90) [inline flow/src/main/java/com/sequenceiq/flow/domain/FlowChainLog.java:96]: Please discuss with @bergerdenes about this.

Comment (sodre90) [reply flow/src/main/java/com/sequenceiq/flow/domain/FlowChainLog.java:96]: Anyway it looks good to me!

**Review by sodre90 — APPROVED** [review]

File: flow/src/main/java/com/sequenceiq/flow/domain/FlowChainLog.java (hunk 1 above)
Comment (bergerdenes) [reply flow/src/main/java/com/sequenceiq/flow/domain/FlowChainLog.java:96]: Please use similar double deserialization as it is present in the `FlowChainHandler.restoreFlowChain`

**Review by bergerdenes — CHANGES_REQUESTED** [review]

File: flow/src/main/java/com/sequenceiq/flow/domain/FlowChainLog.java (hunk 1 above)
Comment (foldik) [reply flow/src/main/java/com/sequenceiq/flow/domain/FlowChainLog.java]: I put that double deserialization into a method in FlowChainLog.

**Review by bergerdenes — APPROVED** [review]

**Pull request merged**